### DISCUSSION TITLE
is it posiible to get the summary data of a general extraction

### DISCUSSION THREAD

**Discussion by mrpotatoserver** [pitchmuc/adobe_analytics_api_2.0#58] [Ideas]
I'm trying to retrieve the SummaryData values  comprehending the Totals of a report with two metrics. I it possible to retrieve these totals?
It's very interesting to be able to retrieve these values in order to query the information that Adobe provides. Can these values consulted directly with the "getReport" method?

To clarify what I'm asking, when I retrieve the JSON file of the freeform table at the very end I get the summary data values that I want to query:

```
"summaryData": {
        "filteredTotals": [
            3565441,
            465168
        ],
        "totals": [
            3565441,
            465168
        ],
        "col-max": [
            2564,
            65411
        ],
        "col-min": [
            1,
            1
        ]
```

**Comment by pitchmuc** [comment]
Hello MrPotato

At the moment, I am not showing this number. 
I will try to include it in one of the future release. 

I will let you know then.
